examples/Spec/IntCompact.hs

```diff
@@ -29,13 +31,13 @@ instance Description IntDescr Int where
         | i < 11 && i > -111 = Small
         | otherwise = Large {isBound = i == minBound || i == maxBound}
 
-  descriptionGen Zero = pure 0
-  descriptionGen (Positive (Large True)) = pure maxBound
-  descriptionGen (Positive (Large False)) = int (linear 11 (maxBound - 1))
-  descriptionGen (Positive Small) = int (linear 1 10)
-  descriptionGen (Negative (Large True)) = pure minBound
-  descriptionGen (Negative (Large False)) = int (linear (minBound + 1) (-11))
-  descriptionGen (Negative Small) = int (linear (-10) (-1))
+  genForDescription Zero = pure 0
```

Comment:
```haskell
descriptionGen = \case
  Zero -> pure 0
  Positive (Large True) -> pure maxBound
  ..
```
is a lot easier on the eyes

Reply:
The autoformatter doesn't like custom alignment :confused: